runtime/lua/vim/fs.lua

```diff
@@ -142,16 +143,16 @@ function M.dir(path, opts)
   end)
 end
 
---- Find files or directories in the given path.
+--- Find any items in the given path.
```

Comment:
For this one case, it makes sense to be more concrete. Also "any items" sounds weird here.
```suggestion
--- Find files or directories (or other items as specified by `opts.type`) in the given path.
```

Reply:
Thanks for pointing this out, but can I change <code>\`opts.type\`</code> to `{opts.type}`?

Reply:
Sure, but I'm not sure that pattern is something I want to see all over the docs. The `{foo}` thing really stands out, and it can get noisy compared to codespans.

Reply:
I see.